### ISSUE TITLE
Empty View is not possible?

### ISSUE THREAD

**Issue by Jogala** [getml/getml-community#13]
say I create a getml split via
```python
split = getml.data.split.random(train=0.8, test=0.2, seed=42)
```

hence, as no validation set was set, running

```python
validation = target_table_pipeline[split == "validation"]
```
should lead to an empty getml.data.view.View. 

However, e.g., 

```
print(validation)
```

leads to the error 

```
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
Cell In[11], line 1
----> 1 print(pipeline.target_table_split.validation)

File ~/.venv/lib/python3.11/site-packages/getml/data/view.py:281, in View.__repr__(self)
    280 def __repr__(self):
--> 281     formatted = self._format()
    283     footer = self._collect_footer_data()
    285     return formatted._render_string(footer=footer)

File ~/.venv/lib/python3.11/site-packages/getml/data/view.py:187, in View._format(self)
    186 def _format(self):
--> 187     return _ViewFormatter(self)

File ~/.venv/lib/python3.11/site-packages/getml/utilities/formatting/view_formatter.py:96, in _ViewFormatter.__init__(self, view, num_head)
     93 if self.subroles:
     94     index_headers += ["subroles:"] + [f"- {cat:7}" for cat in self.subroles]
---> 96 self._add_index(index_headers)

File ~/.venv/lib/python3.11/site-packages/getml/utilities/formatting/view_formatter.py:103, in _ViewFormatter._add_index(self, headers)
    100 def _add_index(self, headers):
    101     index_column = _ViewFormatColumn(
    102         headers=headers,
--> 103         cells=list(range(len(self.data[0]))),
...
    105         n_cells=self.n_rows,
    106     )
    108     self.data = [index_column] + self.data

IndexError: list index out of range
```

**Comment by srnnkls** [comment]
This will be fixed in the next release, 1.5, which we will be releasing in the next few days. I'll keep the issue open until 1.5 is released.

**Comment by srnnkls** [comment]
1.5.0 is released now.
